README.md

```diff
@@ -176,7 +176,12 @@ Pwwka.configure do |config|
 end
 ```
 
-Regardless of which processor you use, the name of the queue created is `pwwka_send_message_async`.
+Regardless of which processor you use, the name of the queue created is `pwwka_send_message_async`. You will need to start a worker process to work the queue. For a `Procfile` setup, with Resque as the processor, that could look something like this:
```

Comment:
Given that this is open source, it's possible that users don't use foreman/`Procfile`s. It's probably sufficient here to simply give the rake task words and let users translate that into whatever job runner they use in their ops environment.

Reply:
Yeah, I thought about that. Other parts of the Readme phrase it similarly to this, where they aren't saying something as definitive as "put this in your Procfile", but more like "if you have a Procfile, here is an example of what it could look like".

Reply:
I think this nicely matches the wording of the other Procfile examples.  Thanks for the update!